src/main/java/by/custom/utilcalculator/domain/UserProgress.java

```diff
@@ -12,17 +14,23 @@ public class UserProgress implements Serializable {
     private VolumeOfEngine volumeOfEngine = null;
     private final String chatID;
     private Step currentQuestion;
+    private Enum[] stepsData;
```

Comment:
I believe this could be configured based on above data

Reply:
Yes, sorry